modular-docs-manual/content/topics/assembly_creating_procedure_modules.adoc

```diff
@@ -0,0 +1,13 @@
+[[creating-procedure-modules]]
+= Creating Procedure Modules
+
+This section explains what a procedure module is and provides recommended practices for writing procedure modules.
+
+include::module_definition-procedure.adoc[leveloffset=+1]
+
+include::module_guidelines-procedure.adoc[leveloffset=+1]
+
+== Related Information
+
+* Download the link:https://gitlab.cee.redhat.com/ccs-internal-documentation/Modular_Documentation_Project/raw/master/files/TEMPLATE_PROCEDURE_doing_one_procedure.adoc[procedure module template] for new projects.
```

Comment:
I would suggest using the file name for the link title to make it immediately clear that this is a download of an adoc file. Like this:

`link:https://gitlab.cee.redhat.com/ccs-internal-documentation/Modular_Documentation_Project/raw/master/files/TEMPLATE_PROCEDURE_doing_one_procedure.adoc[TEMPLATE_PROCEDURE_doing_one_procedure.adoc]`